### ISSUE TITLE
If typographer enabled, quoted values don't work

### ISSUE THREAD

**Issue by rantecki** [arve0/markdown-it-attrs#11]
When MarkdownIt is run with `{ typographer: true}` to enable typographic replacements, quotes (`"`) are replaced with `“”`.  This seems to break the attribute parser.

E.g. For this test content:
```
Test{key1="a value with spaces"}
```

With typographer disabled, the result is correct:
```
<p key1="a value with spaces">Test</p>
```

With typographer enabled, the result is:
```
<p key1="“a" value="" with="" spaces”="">Test</p>
```

**Comment by arve0** [comment]
Thanks for reporting.

**Comment by arve0** [comment]
https://github.com/arve0/markdown-it-attrs/blob/master/utils.js#L47

**Comment by arve0** [comment]
Fix released in `v0.3.0`. Let me know if it doesn't work for you.

**Comment by rantecki** [comment]
Looks good thanks!
